woocommerce/woocommerce-android
# [Linked Products] Product Selection Issues

Closes: #10815 


### Description

With this PR we fix two issues on the product selection fragment when linking other products to a product:
- The Done button wasn't visible on Light mode. This is fixed by ~~adding the assets in light (black asset) and night (white) mode~~ using the purple check button, keeping the same color as in other places.
- The barcode scanner button doesn't react. This is a side effect of using the menu product list fragment that contains the barcode scanner button to update inventory. Inflating the menu with the `menu_product_selection_fragment` instead fixes the issue.

### Testing instructions


1. Go to Products
2. Select a product
3. Tap on Add More Details
4. Tap on Linked Products
5. Tap on Add Products
6. Tap on Add Product
7. See that the Barcode Scanner Button is not visible
8. Select a Product
9. See that the done check button is visible on light and night themes. See that it reacts properly to the tap.

### Images/gif


#### Before
https://github.com/woocommerce/woocommerce-android/assets/4923871/1a8c34e6-c4dc-47dc-8bff-cc65654d6482

#### After

https://github.com/woocommerce/woocommerce-android/assets/1864060/5660577b-dc3a-418c-9c13-832059949279
